t/openssl/x509/csr.lua

```diff
@@ -1,53 +1,52 @@
-
 local function create_csr(domain_pkey, ...)
-  local domains = {...}
-
-  local subject = require("resty.openssl.x509.name").new()
-  local _, err = subject:add("CN", domains[1])
-  if err then
-    return nil, err
-  end
+    local domains = { ... }
```

Comment:
please avoid style change here, we have 2 space indention in this project

Reply:
Reverted